README.md

```diff
@@ -861,6 +861,28 @@ func main() {
 	r.Run(":8080")
 }
 ```
+#### JSONP
```

Comment:
Can you also add one line at top of `API Examples section`?

Reply:
Ok, I have added documentation.